src/main/java/org/opentripplanner/graph_builder/module/osm/DefaultWayPropertySetSource.java

```diff
@@ -37,645 +43,394 @@
  */
 public class DefaultWayPropertySetSource implements WayPropertySetSource {
 
+  private final WayPropertiesBuilder allWayProperties = withModes(ALL);
```

Comment:
It might make sense to build these, so nobody tries to re-use these. `setProperties` could have an override with the builder, which just builds and passes the resulting object to the regular method, which takes a built object